org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/adapters/PrepareRenameResponseAdapter.java

```diff
@@ -1,5 +1,5 @@
 /******************************************************************************
- * Copyright (c) 2018 Microsoft Corporation and others.
+ * Copyright (c) 2019 Microsoft Corporation and others.
```

Comment:
If you update the year in the file header, it should be a range, e.g. `2018-2019`.

https://www.eclipse.org/projects/handbook/#ip-copyright-headers

> the first year is when the content was first created and the last year is when the content was last modified.